yarn.lock

```diff
@@ -11176,10 +11176,10 @@ execa@^5.0.0:
     signal-exit "^3.0.3"
     strip-final-newline "^2.0.0"
 
-exenv-es6@^1.0.0:
-  version "1.0.0"
-  resolved "https://registry.yarnpkg.com/exenv-es6/-/exenv-es6-1.0.0.tgz#bd459136369af17cf33f959b5af58803d4068c80"
-  integrity sha512-fcG/TX8Ruv9Ma6PBaiNsUrHRJzVzuFMP6LtPn/9iqR+nr9mcLeEOGzXQGLC5CVQSXGE98HtzW2mTZkrCA3XrDg==
+exenv-es6@^1.1.0, exenv-es6@^1.0.0:
```

Comment:
Is something still hanging on to 1.0.0? I'm curious why both versions are here

Reply:
That reference is coming from the deprecated/archived React packages which are dependencies of the component explorer.